portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,33 @@
+body {
+    background-color: #2e5599;
+    background-image: url("https://www.transparenttextures.com/patterns/black-scales.png");
```

Comment:
I'd say you should download this image and serve it up yourself rather than hitting their servers for it. That will prevent any issues with their site from messing with your site. It's also polite to not have traffic to your site causing traffic to someone else's site. That's not really a rule, more just a guideline.

Reply:
Normally I download the image when I'm done developing as I'm pretty indecisive and can change my mind on things like the background fairly often. I never thought about the traffic on my site affecting their site, good point. I downloaded the image and changed the style.css file.